en/docs/InstallAndSetup/DeployingWSO2APIManager/SingleNode/all-in-one-deployment-overview.md

```diff
@@ -1,24 +1,14 @@
 # All-in-One Deployment Overview
 
-In a typical production deployment, API Manager is deployed as components (Publisher, Store, Gateway, Key Manager and Traffic Manager). While this provides very high performance and a high level of scalability, it may be too complex if you want to run API Manager as a small to medium scale API Management solution. A WSO2 API-M single node deployment, which has all the API-M components in one instance, would be simple to set up and requires less resources when compared with a distributed deployment. It is ideal for any organization that wants to start small and iteratively build up a robust API Management Platform.
+In a typical production deployment, API Manager is deployed as components (Publisher, Store, Gateway, Key Manager 
```

Comment:
Store should be replaced with the Dev Portal

Reply:
Fixed